pkg/cri/sbserver/sandbox_stats_linux.go

```diff
@@ -162,11 +163,11 @@ func metricsForSandbox(sandbox sandboxstore.Sandbox) (interface{}, error) {
 		statsx = stats
 
 	} else {
-		control, err := cgroups.Load(cgroups.V1, cgroups.StaticPath(cgroupPath))
+		control, err := cgroup1.Load(cgroup1.Default, cgroup1.StaticPath(cgroupPath))
```

Comment:
Let me know your thoughts about this subtle renaming. I didn't like cgroup1.V1 and renamed that to cgroup1.Default.

Reply:
Maybe you can just drop the first arg